<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>683</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="topLayout">
      <property name="spacing">
       <number>0</number>
      </property>
     </layout>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="minimumSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="playlistPage">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QTabWidget" name="tabs">
           <property name="acceptDrops">
            <bool>true</bool>
           </property>
           <property name="tabPosition">
            <enum>QTabWidget::West</enum>
           </property>
           <property name="currentIndex">
            <number>-1</number>
           </property>
          </widget>
          <widget class="QWidget" name="verticalLayoutWidget">
           <layout class="QVBoxLayout" name="currentListLayout"/>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="shoutcastPage">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QSplitter" name="scSplitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QGroupBox" name="genreGroup">
           <property name="title">
            <string>Shoutcast Genres</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_6">
            <property name="margin">
             <number>2</number>
            </property>
            <item row="4" column="0" colspan="3">
             <widget class="QTreeWidget" name="genreList">
              <property name="contextMenuPolicy">
               <enum>Qt::ActionsContextMenu</enum>
              </property>
              <property name="dragEnabled">
               <bool>true</bool>
              </property>
              <property name="animated">
               <bool>true</bool>
              </property>
              <attribute name="headerVisible">
               <bool>false</bool>
              </attribute>
              <column>
               <property name="text">
                <string>Genres</string>
               </property>
              </column>
             </widget>
            </item>
            <item row="5" column="0" rowspan="2" colspan="3">
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <property name="topMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QPushButton" name="reloadGenres">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Reload</string>
                </property>
                <property name="icon">
                 <iconset>
                  <normaloff>icons/view-refresh.png</normaloff>icons/view-refresh.png</iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="saveStation">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Bookmark</string>
                </property>
                <property name="icon">
                 <iconset>
                  <normaloff>icons/document-save-as.png</normaloff>icons/document-save-as.png</iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="previewStation">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Preview</string>
                </property>
                <property name="icon">
                 <iconset>
                  <normaloff>icons/media-playback-start.png</normaloff>icons/media-playback-start.png</iconset>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="0" column="0" rowspan="2" colspan="3">
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <property name="topMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Search</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="KLineEdit" name="scSearch">
                <property name="showClearButton" stdset="0">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QGroupBox" name="bookmarksGroup">
           <property name="title">
            <string>Shoutcast Bookmarks</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_7">
            <property name="margin">
             <number>2</number>
            </property>
            <item row="0" column="0">
             <widget class="QListWidget" name="bookmarkList">
              <property name="contextMenuPolicy">
               <enum>Qt::ActionsContextMenu</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Plain</enum>
              </property>
              <property name="dragDropMode">
               <enum>QAbstractItemView::DragDrop</enum>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::SingleSelection</enum>
              </property>
              <property name="selectionRectVisible">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>599</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuConnect">
    <property name="title">
     <string>&amp;Connect to</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuMPD">
    <property name="title">
     <string>&amp;MPD</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuConnect"/>
   <addaction name="menuMPD"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="icon">
    <iconset>
     <normaloff>icons/application-exit.png</normaloff>icons/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset>
     <normaloff>icons/configure.png</normaloff>icons/configure.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="currentMenuPlay">
   <property name="icon">
    <iconset>
     <normaloff>icons/media-playback-start.png</normaloff>icons/media-playback-start.png</iconset>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
   <property name="toolTip">
    <string>Play song</string>
   </property>
  </action>
  <action name="currentMenuRemove">
   <property name="icon">
    <iconset>
     <normaloff>icons/list-remove.png</normaloff>icons/list-remove.png</iconset>
   </property>
   <property name="text">
    <string>Remove from list</string>
   </property>
   <property name="toolTip">
    <string>Remove song from the playlist.</string>
   </property>
  </action>
  <action name="currentMenuClear">
   <property name="icon">
    <iconset>
     <normaloff>icons/page-simple.png</normaloff>icons/page-simple.png</iconset>
   </property>
   <property name="text">
    <string>Clear list</string>
   </property>
   <property name="toolTip">
    <string>Clear the current playlist.</string>
   </property>
  </action>
  <action name="currentMenuSave">
   <property name="icon">
    <iconset>
     <normaloff>icons/document-save-as.png</normaloff>icons/document-save-as.png</iconset>
   </property>
   <property name="text">
    <string>Save playlist</string>
   </property>
   <property name="toolTip">
    <string>Save the current playlist.</string>
   </property>
  </action>
  <action name="currentMenuCrop">
   <property name="icon">
    <iconset>
     <normaloff>icons/project-development-close.png</normaloff>icons/project-development-close.png</iconset>
   </property>
   <property name="text">
    <string>Crop Playlist</string>
   </property>
   <property name="toolTip">
    <string>Remove all but the selected songs from the playlist.</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>genreList</tabstop>
  <tabstop>bookmarkList</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
